canopy_driver.F90

```diff
@@ -100,9 +100,9 @@ program canopy_driver
       zkcm   = profile%zk 
       ztothc = zkcm/hcm 
       resz   = profile%dzk
-      cansublays  = ceiling(hcm/resz(canlays))
+      cansublays  = floor(hcm/resz(canlays))
       canmidpoint = cansublays/2
-      flamelays    = ceiling(flameh/resz(canlays))
```

Comment:
@zmoon shouldn't this also then just be nint((hcm/resz(canlays)), same as in flamelays?

Reply:
well it could round up, and then it would be above the canopy height (`HCM`)

Reply:
@zmoon We want floor for both cansublays and flamelays, or just leave them as is to truncate...

Reply:
I only changed it all because it was generating compiler warnings due to the type conversion

Reply:
will change flamelays to floor as well